ohara-manager/utils/configHelpers.js

```diff
@@ -43,8 +43,29 @@ const validateUrl = async url => {
 
   // Ensure the API URL can be reach
```

Comment:
Oops, my own typo...
```suggestion
  // Ensure the API URL can be reached
```

Reply:
had been edited :)